rust/src/client/mobile_client/participant/mod.rs

```diff
@@ -0,0 +1,83 @@
+//! Provides the logic and functionality for a participant of the PET protocol.
+//!
+//! See the [client module] documentation since this is a private module anyways.
+//!
+//! [client module]: ../index.html
+use crate::{
+    certificate::Certificate,
+    mask::config::MaskConfig,
+    message::message::{MessageOwned, MessageSeal},
+    CoordinatorPublicKey,
+    ParticipantPublicKey,
+    ParticipantSecretKey,
+};
+use derive_more::From;
+
+pub mod sum;
+pub mod sum2;
+pub mod undefined;
+pub mod update;
+
+pub use self::{sum::Sum, sum2::Sum2, undefined::Undefined, update::Update};
+
+pub struct ParticipantState {
+    // credentials
+    pub pk: ParticipantPublicKey,
```

Comment:
Maybe we could use the `EncryptKeyPair` for this now.

Reply:
`SigningKeyPair`

Reply:
Ah true, I will change it

Reply:
Shall we do the same with        

```
ephm_pk: SumParticipantEphemeralPublicKey,
ephm_sk: SumParticipantEphemeralSecretKey,
```

in `Participant<Sum>` and `Participant<Sum2>` as well?